Tests/SWBWindowsPlatformTests/SWBWindowsPlatformTests.swift

```diff
@@ -9,4 +9,69 @@
+    func ldMacroTable() async throws ->  MacroValueAssignmentTable {
+            let core = try await getCore()
+            return MacroValueAssignmentTable(namespace: core.specRegistry.internalMacroNamespace)
+    }
+
+    @Test(.requireSDKs(.windows))
+    func discoveredLdLinkerSpecInfo_Windows() async throws {
+        var table = try await ldMacroTable()
+        let core = try await getCore()
+        table.push(BuiltinMacros._LD_MULTIARCH, literal: true)
+        try await withSpec(LdLinkerSpec.self, .deferred, platform: "windows", additionalTable: table) { (info: DiscoveredLdLinkerToolSpecInfo) in
+            #expect(!info.toolPath.isEmpty)
+            #expect(info.toolVersion != nil)
+            if let toolVersion = info.toolVersion {
+                #expect(toolVersion > Version(0, 0, 0))
+            }
+            #expect(info.linker == .lld)
+        }
+        try await withSpec(LdLinkerSpec.self, .result(status: .exit(0), stdout: Data("Microsoft (R) Incremental Linker Version 14.41.34120.0\n".utf8), stderr: Data()), platform: "windows", additionalTable: table) { (info: DiscoveredLdLinkerToolSpecInfo) in
+            #expect(!info.toolPath.isEmpty)
+            #expect(info.toolVersion == Version(14, 41, 34120))
+            #expect(info.architectures == Set())
+        }
+
+        // link.exe cannot be used for multipler architectures and requires a distinct link.exe for each target architecture
+        table.push(BuiltinMacros.ALTERNATE_LINKER, literal: "link")
+        table.push(BuiltinMacros._LD_MULTIARCH, literal: false)
+        table.push(BuiltinMacros._LD_MULTIARCH_PREFIX_MAP, literal: ["x86_64:x64", "aarch64:arm64", "i386:x86", "thumbv7:arm"])
+
+        // x86_64
+        if try await core.hasVisualStudioComponent(.visualCppBuildTools_x86_x64) {
+            table.push(BuiltinMacros.ARCHS, literal: ["x86_64"])
+            try await withSpec(LdLinkerSpec.self, .deferred, platform: "windows", additionalTable: table) { (info: DiscoveredLdLinkerToolSpecInfo) in
+                #expect(!info.toolPath.isEmpty)
+                #expect(info.toolVersion != nil)
+                if let toolVersion = info.toolVersion {
+                    #expect(toolVersion > Version(0, 0, 0))
+                }
+                #expect(info.linker == .linkExe)
+            }
+        }
+
+        // link aarch64
+        if try await core.hasVisualStudioComponent(.visualCppBuildTools_arm64) {
+            table.push(BuiltinMacros.ARCHS, literal: ["aarch64"])
+            try await withSpec(LdLinkerSpec.self, .deferred, platform: "windows", additionalTable: table) { (info: DiscoveredLdLinkerToolSpecInfo) in
+                #expect(!info.toolPath.isEmpty)
+                #expect(info.toolVersion != nil)
+                if let toolVersion = info.toolVersion {
+                    #expect(toolVersion > Version(0, 0, 0))
+                }
+                #expect(info.linker == .linkExe)
+            }
+        }
+    }
```

Comment:
Can we add armv7 (visualCppBuildTools_arm) and arm64ec (visualCppBuildTools_arm64ec) blocks here as well? Then we'll have coverage of all 5 ABIs.

arm64ec will map to arm64 in the linker arch map.